packages/back-end/src/models/UsageAlertsModel.ts

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+interface UsageAlertInterface {
+  id: string;
+  percentUsed: number;
+  orgId: string;
+  timeframeEnd: Date;
+  meterName: string;
+  dateAdded: Date;
+}
+
+const usageAlertSchema = new mongoose.Schema({
+  id: {
+    type: String,
+    unique: true,
```

Comment:
Added `unique: true` so if we try to add a webhook id that's already been processed, it'll fail gracefully with a 200.